lib/pure/strformat.nim

```diff
@@ -94,7 +94,7 @@ The general form of a standard format specifier is::
 
   [[fill]align][sign][#][0][minimumwidth][.precision][type]
 
-The brackets ([]) indicate an optional element.
+The square brackets ``[]`` indicate an optional element.
```

Comment:
Given that both () and [] are called "brackets" in different dialects, it seemed wrong to use them together when attempting to clarify the word.

Reply:
I believe `[]` are called just *brackets* and `()` are called *parentheses*.

Reply:
https://en.oxforddictionaries.com/punctuation/parentheses-and-brackets

Reply:
In UK English `()` are called brackets https://separatedbyacommonlanguage.blogspot.com/2006/08/parentheses-and-brackets.html

Reply:
oh well, OK :)